src/Parser/Shaman/Restoration/CONFIG.js

```diff
@@ -1,15 +1,26 @@
-import { Versaya } from 'MAINTAINERS';
+import React from 'react';
+import { Hartra344, Versaya } from 'MAINTAINERS';
 import SPECS from 'common/SPECS';
 import SPEC_ANALYSIS_COMPLETENESS from 'common/SPEC_ANALYSIS_COMPLETENESS';
-
 import CombatLogParser from './CombatLogParser';
 import CHANGELOG from './CHANGELOG';
 
 export default {
   spec: SPECS.RESTORATION_SHAMAN,
-  maintainers: [Versaya],
-  completeness: SPEC_ANALYSIS_COMPLETENESS.NOT_ACTIVELY_MAINTAINED, // good = it matches most common manual reviews in class discords, great = it support all important class features
+  maintainers: [Hartra344, Versaya],
+  completeness: SPEC_ANALYSIS_COMPLETENESS.NEEDS_MORE_WORK,
+  description: (
+    <div>
+      Welcome to the Resto Shaman analyzer! We hope you find these suggestions and statistics useful.
+      <br /><br />
+
+      If you want to learn more about Resto Shaman, join the Druid community at the <a href="https://discord.gg/AcTek6e" target="_blank" rel="noopener noreferrer">Ancestral Guidance channel</a>.
```

Comment:
Typo: "join the Druid community" presumably means Shaman.

Reply:
yep, fixed 👍  I took the format of the resto druid description. Nice and simple